cypress/integration/institution.spec.js

```diff
@@ -40,7 +40,7 @@ describe('My profile: Institutions', () => {
   });
 
   it('The user should be able to remove an insitution from their profile', () => {
-    cy.get('[data-testid=button-delete-institution-194\\.65\\.20\\.10]').click({ force: true });
+    cy.get('[data-testid=button-delete-institution-194\\.65\\.20\\.0]').click({ force: true });
```

Comment:
vet ikke hvorfor, men det virket etter denne endringen. ny struktur på id-er for institusjoner? @sarahse

Reply:
jeg kan ikke skjønne hvorfor det skulle funket? vi bruker jo mockdata? er det endret noe på mockdataen da?

Reply:
nå virker testene lokalt, men ikke her, så det er jo interessant

Reply:
Det er jo denne (https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/utils/testfiles/institutions/unit_response.json#L27) som legges til og fjernes, så det gir jo mening at det skal være id som slutter på `.0`(?)

Reply:
fra travis: "CypressError: Timed out retrying: Expected to find element: '[data-testid=button-delete-institution-194\.65\.20\.0]', but never found it."
kan det være at travis takler escape characters annerledes enn lokalt?

Reply:
rart. jeg tror det er en feil i mock-interceptoren, for når en institusjon blir lagt til i My Profile, så skal det dispatches en action som oppdaterer authority-objektet, som igjen skal føre til en rerendring av My Profile (hvor authority har `orgunitid:[194.65.20.10]`. Som blir kalt her:
`mock.onPut(new RegExp(`${API_URL}${AuthorityApiPaths.AUTHORITY}/*`)).reply(200, mockSingleAuthorityResponseWithOrcid);`

Men ser at det også er en feil i denne testen, for det neste nivået skulle også egentlig bli satt. slik at id'en som ble lagret skulle vært `194.65.20.10`. Jeg kan sjekke litt hvis du vil?

Reply:
evt kan du kommentere ut denne testen helt til vi får koblet på backend og ser hvordan det faktisk vil oppføre seg. Jeg regner med at vi uansett må skrive om denne testen.

Reply:
tror jeg fant feilen. det var pga det ble brukt pop istedenfor slice 😅

Reply:
skjønte ingenting av det der, så du kan gjerne se på det hvis du har tid og lyst :P

Reply:
hvis ikke kommenterer jeg bare ut den siste sjekken som feiler